test/index.spec.js

```diff
@@ -1,13 +1,123 @@
-const chai = require('chai')
-const chaiHttp = require('chai-http')
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const Mail = require("../server/helper/mail");
 
-chai.use(chaiHttp)
-const expect = chai.expect
+chai.use(chaiHttp);
+const { expect } = chai;
+chai.should();
 
 const add = 2 + 2;
-describe('Sample Test', () => {
-    it('Add number', () => {
-       expect(add).to.eql(4)
-    })
-    
-})
+describe("Sample Test", () => {
```

Comment:
Let's get rid of this test. Also, let's reorganise this test to match the project: `test/helpers/mail.spec.js`.

Reply:
I resolved it.